### DISCUSSION TITLE
Color Coded Tabs

### DISCUSSION THREAD

**Discussion by KlingonMike** [sumatrapdfreader/sumatrapdf#3856] [Ideas]
Suggestion: Allow the user to right-click on a TAB and color-code it. When you have so many PDFs with similar names as I do, it is extremely difficult to quickly identify one from the other. It would be nice if you could right-click on each individual TAB and select a color for that TAB.

**Comment by GitHubRulesOK** [comment]
hmm
that would probably need a user interface, unless there was a single setting per remembered file with only one default value. then all 8 million users could edit their own colours from 16.7 Million available

**Comment by KlingonMike** [comment]
Well, it already remembers many settings for each PDF, this would be just one more.
